tests/unit/systems/ops/feast/test_op.py

```diff
@@ -105,29 +106,17 @@ def test_feast_from_feature_view(tmpdir, suffix_int):
                 ColumnSchema(name="prefix_int_feature", dtype=np.int32),
                 ColumnSchema(name="prefix_float_feature", dtype=np.float32),
                 ColumnSchema(
-                    name=f"prefix_int_list_feature_{suffix_int}",
```

Comment:
We no longer use suffix integers to identify arrays that go together, since we have the `__values`/`__lengths` convention.